pkg/schema/schema.go

```diff
@@ -317,3 +328,80 @@ func validateResources(directory string) error {
 func CheckCatalogResources(manifestDir string) error {
 	return validateResources(manifestDir)
 }
+
+func validateExamplesAnnotations(csv *v1alpha1.ClusterServiceVersion) error {
+	var examples []CRDTemplate
+	annotations := csv.ObjectMeta.GetAnnotations()
+	// Return right away if no alm-examples are found.
+	if annotations == nil {
+		return fmt.Errorf("Unable to get annotations")
+	}
+
+	almExamples, ok := annotations["alm-examples"]
```

Comment:
It might be nice to go ahead and support both `alm-examples` and `olm.examples` or something, so that we can start to phase out the ALM name.